superset-frontend/src/components/ListView/ListViewStyles.less

```diff
@@ -104,8 +104,16 @@
     }
 
     .table-row {
+      .actions {
+        opacity: 0;
```

Comment:
Awwww, didn't like the animation?
`transition: opacity @timing-normal;`
^I'm tempted to add this kind of thing lots of places in Superset ;)

Reply:
Done!